website/content/partials/known-issues/1_15-audit-file-sighup-does-not-trigger-reload.mdx

```diff
@@ -0,0 +1,42 @@
+### File audit devices do not honor SIGHUP signal to reload
+
+#### Affected versions
+
+- 1.15.0
+
+#### Issue
+
+The new underlying event framework for Vault causes Vault to continue using 
+audit logs instead of reopening the file paths even when you send 
+[`SIGHUP`](/vault/docs/audit/file#log-file-rotation) after log rotation:
+
+- If you move or rename your audit log file, Vault continues to log data to the
+  moved file. For example, if you archive the log file
+  `audit/log/path/vault-audit.log` by moving it to
+  `audit/log/archive/path/vault-audit.log.bak`, Vault continues to write data to
+  `audit/log/archive/path/vault-audit.log.bak` instead of starting a fresh log
+  at `audit/log/path/vault-audit.log`.
+- If you delete your audit log file, Vault continues to write data to the
+  deleted file, which will continue to consume disk space as it grows. When
+  Vault is sealed or restarted, the OS will perform junk collection on the
+  deleted file and you will lose all data logged the audit file after it was
+  tagged for deletion.
+
+<Warning title="Missing Audit Entries">
+  If the file was deleted (rather than moved locally) then audit entries written
+  by Vault will be lost.
+</Warning>
```

Comment:
```suggestion
<Warning title="You may lose audit entries">
  If you delete the audit file on rotation, rather than moving it, you
  will lose any entries written by Vault after rotation when the OS
  permanently deletes the file.
</Warning>
```